.github/workflows/snyk.yml

```
@@ -17,16 +17,16 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Run snyk test to check for vulnerabilities
-        if: github.ref != 'ref/head/main'
+        if: github.ref != 'refs/heads/main'
         uses: snyk/actions/scala@0.3.0
         env:
           SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
 
       - name: Run snyk monitor and report vunerabilites to snyk.io
-        if: github.ref == 'ref/head/main'
+        if: github.ref == 'refs/heads/main'
         uses: snyk/actions/scala@0.3.0
         env:
           SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
         with:
-          args: --org=the-guardian-cuu --project-name=guardian/manage-frontend
+          args: --org=the-guardian-cuu --project-name=guardian/support-service-lambdas
```

According to [this](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context), the repo name is exposed as an env var.  So this might work:
```suggestion
          args: --org=the-guardian-cuu --project-name=${{github.repository}}
```